### PR TITLE
docs(nodes): refactor doc strings for read the docs

### PR DESCRIPTION
**Pull request by ongzizhao**
Refactored the doc strings to describe pipeline , live node, recorded node and node

### CODE REVIEW

File: peekingduck/pipeline/nodes/input/live.py [hunk 1]
```diff
@@ -24,7 +24,47 @@
...
+
+        |pipeline_end|
+
+        |saved_video_fps|
+
+    Configs:
+        fps_saved_output_video (:obj:`int`): **default = 10**
+
+            FPS of the mp4 file after livestream is processed and exported.
+            FPS dependant on running machine performance.
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/input/live.py:45]: should be `dependent` instead of `dependant`

File: peekingduck/pipeline/nodes/input/live.py [hunk 2]
```diff
@@ -24,7 +24,47 @@
...
+
+            Filename of the mp4 file if livestream is exported.
+
+        resize (:obj:`Dict`): **default = { do_resizing: False, width: 1280, height: 720 }**
+
+            Dimension of extracted image frame
+
+        input_source (:obj:`int`): **default = 0 (for webcam)**
+
+            Refer to OpenCV doucmentation for list of source stream codes
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/input/live.py:57]: link to the relevant page in OpenCV using 
```suggestion
            Refer to `OpenCV doucmentation <https//www.google.com>`_ for list of source stream codes
```

File: peekingduck/pipeline/nodes/input/live.py [hunk 3]
```diff
@@ -24,7 +24,47 @@
...
+            FPS of the mp4 file after livestream is processed and exported.
+            FPS dependant on running machine performance.
+
+        filename (:obj:`str`):  **default = "webcamfeed.mp4"**
+
+            Filename of the mp4 file if livestream is exported.
+
+        resize (:obj:`Dict`): **default = { do_resizing: False, width: 1280, height: 720 }**
+
+            Dimension of extracted image frame
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/input/live.py:53]: use the same definition as `recorded.py`

File: peekingduck/pipeline/nodes/input/recorded.py [hunk 4]
```diff
@@ -24,7 +24,34 @@
...
+
+                User defined dimension of extracted image frame
+
+            input_dir (:obj: `str`): **default = 'PeekingDuck/data/input'**
+
+                The directory to look for recorded video files and images
+
+            mirror_image (:obj:`bool`): **default = False**
+
+                User's choice to set extracted image frame as mirror image of input stream
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/input/recorded.py:52]: use the same definition as `live.py`

File: peekingduck/pipeline/pipeline.py [hunk 5]
```diff
@@ -23,14 +23,17 @@
 
 class Pipeline:
     """ Pipe class that stores nodes and manages the data information used during inference
```
Comment (sidney-tio) [inline peekingduck/pipeline/pipeline.py:25]:
```suggestion
    """ Pipe class that stores nodes and manages flow of data used during inference
```

**Review by sidney-tio — COMMENTED** [review]
language can be tighter and consistent. thanks!

File: peekingduck/pipeline/nodes/input/live.py (hunk 3 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/live.py:53]: I will stick to "Dimension of extracted image frame" for both live and recorded

File: peekingduck/pipeline/nodes/input/recorded.py (hunk 4 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/recorded.py:52]: i will stick to "Boolean to set extracted image frame as mirror image of input stream" for both live and recorded

File: peekingduck/pipeline/nodes/input/live.py (hunk 2 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/live.py:57]: Thanks. I have made the changes

File: peekingduck/pipeline/nodes/input/live.py (hunk 1 above)
Comment (ongzizhao) [reply peekingduck/pipeline/nodes/input/live.py:45]: Thanks. I have made the changes

**Comment by sidney-tio** [comment]
thanks! LGTM

**Pull request merged**